1.34 prep (#185)
nginx/unit-docs: source/configuration/index.rst

       }
   }

===============
JSON log format
===============

> Starting with NGINX Unit 1.34.0, **format** can instead be an object

Author: I think so, `format` is referenced right above this and then immediately below...

> describing JSON field name/value pairs, e.g.,

Reviewer: ```suggestion
describing JSON field name/value pairs, for example,
```

.. code-block:: json

  {
      "access_log": {
          "path": "/tmp/access.log",
          "format": {
              "remote_addr": "$remote_addr",
              "time_local": "$time_local",
              "request_line": "$request_line",
              "status": "$status",
              "body_bytes_sent": "$body_bytes_sent",
              "header_referer": "$header_referer",
              "header_user_agent": "$header_user_agent"
          }
      }
  }

> The JSON *values* support being strings, variables and JavaScript.

Reviewer: ```suggestion
The JSON *values* can be strings, variables, and JavaScript.
```
